livre/eq/Channel.cpp

```diff
@@ -275,16 +275,9 @@ class Channel
         _renderViewPtr->setViewport( viewport );
 
         livre::Window* window = static_cast< livre::Window* >( _channel->getWindow( ));
```

Comment:
I could not see a difference. generateRenderingSet() is done again in the if(dashTreeUpdate), so I removed this. I'll test a bit more unless I miss some side effect here which you know better than me.